datafusion/expr/src/expr.rs

```diff
@@ -840,6 +840,12 @@ impl WindowFunction {
     }
 }
 
+/// Find DataFusion's built-in window function by name.
+#[deprecated(since = "44.0.0", note = "built-in window functions have been removed")]
```

Comment:
Shall we give a hint about the best replacement API?

Reply:
I'll have to research what the replacement API is first, but yes, that would be good. Once I have Comet tests passing, I will come back and improve this.